### PR TITLE
feat(python): presence sensing message definitions

### PR DESCRIPTION
**Pull request by amitlissack**

### CODE REVIEW

File: python/opentrons_ot3_firmware/messages/payloads.py [hunk 1]
```diff
@@ -139,3 +139,12 @@ class ReadMotorDriverRegisterResponsePayload(utils.BinarySerializable):
 
     reg_addr: utils.UInt8Field
     data: utils.UInt32Field
+
+
+@dataclass
+class ReadPresenceSensingVoltageResponsePayload(utils.BinarySerializable):
+    """Read head presence sensing voltage response payload."""
+
+    z_motor: utils.UInt16Field
```
Comment (amitlissack) [inline python/opentrons_ot3_firmware/messages/payloads.py:148]: @sotaoverride is uint 16 the correct type for these attributes?

Comment (sfoster1) [inline python/opentrons_ot3_firmware/messages/payloads.py:148]: should probably note the unit here - mV? adc least-significant-bits?

**Review by sfoster1 — APPROVED** [review]
Looks good but we should set what the units are

File: python/opentrons_ot3_firmware/messages/payloads.py (hunk 1 above)
Comment (amitlissack) [reply python/opentrons_ot3_firmware/messages/payloads.py:148]: @sotaoverride ?

Comment (sotaoverride) [reply python/opentrons_ot3_firmware/messages/payloads.py:148]: let me confirm the bits about the units here

Comment (sotaoverride) [reply python/opentrons_ot3_firmware/messages/payloads.py:148]: going off of this sheet[ ](https://docs.google.com/spreadsheets/d/1DMPWo-VjyG0ozShu76oXK-0zwqgU-sOrPBd-KJ6WkTI/edit#gid=785336481) utils.UInt16Field should work fine

Comment (amitlissack) [reply python/opentrons_ot3_firmware/messages/payloads.py:148]: And what are the units?

Comment (sotaoverride) [reply python/opentrons_ot3_firmware/messages/payloads.py:148]: ADC readings were in mV, this sheet shows the max.min values[ ](https://docs.google.com/spreadsheets/d/1DMPWo-VjyG0ozShu76oXK-0zwqgU-sOrPBd-KJ6WkTI/edit#gid=785336481)

Comment (sotaoverride) [reply python/opentrons_ot3_firmware/messages/payloads.py:148]: https://docs.google.com/spreadsheets/d/1DMPWo-VjyG0ozShu76oXK-0zwqgU-sOrPBd-KJ6WkTI/edit#gid=785336481

**Pull request merged**